docs/man/commands/cache.rst

```diff
@@ -0,0 +1,20 @@
+:orphan:
```

Comment:
The same kind of basic file would also be needed in https://github.com/pypa/pip/tree/master/docs/html/reference